src/chocolatey.resources/helpers/functions/Get-Installed.ps1

```diff
@@ -0,0 +1,46 @@
+﻿## Get-Installed
+##############################################################################################################
+## Returns a list of installed applications.
+## Registry based replacement for "Get-WmiObject -Class Win32_Product".
+##############################################################################################################
+
+function Get-Installed {
+param(
+  [string] $NonMSIOption = 'HideNonMSI' # You may use 'ShowNonMSI' option
+)
+
+  if(($NonMSIOption -ine 'HideNonMSI') -and ($NonMSIOption -ine 'ShowNonMSI')) {
+    throw "Unsupported argument: $NonMSIOption"
+  }

```

Comment:
@wjk: thank you!
@ferventcoder: wrapped Get-WmiObject should be compatible with original one.
Would it be better to return non-MSI applications by default and suppress them in Get-WmiObject?